### PR TITLE
Add Swedish translations

### PR DESCRIPTION
**Pull request by maxg0**

### CODE REVIEW

File: shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+msgstr "Betalning för beställning {id} i {shop}"
+
+msgid "We use Stripe for secure payment handling."
+msgstr "Vi använder Stripe för säker betalningshantering."
+
+msgid ""
+"Click the button below to enter your card details. You will only be charged "
+"when your order completes."
+msgstr ""
+"Tryck på nedanstående knapp för att inmata kortets detaljer."
```
Comment (frwickst) [inline shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po]: "inmata" -> "skriva in" ("mata in" is correct as well, but not used as much imho)

File: shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+"Summan debiteras först efter att din beställning är gjord."
+
+#, python-format
+msgid "Accept credit cards using <a target='_blank' href='%(url)s'>Stripe</a>"
+msgstr ""
```
Comment (frwickst) [inline shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po]: The structure of the string does not correspond to the English version.

File: shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+"Godkänn kreditkort med <a target='_blank' href='%(url)s'>Stripe</"
+"a>"
+
+msgid "Don't have a Stripe account?"
+msgstr "Inget Stripe konto?"
+
+msgid "Register now"
+msgstr "Registrera nu"
```
Comment (frwickst) [inline shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po]: "Registrera dig" or "Registrera dig nu" might be better choices here (i prefer the first choice).

File: shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#, python-format
+msgid "Please configure Stripe keys for payment processor %s."
+msgstr "Var god och konfigurera Stripe nycklar för betalningshanterare %s."
```
Comment (frwickst) [inline shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po]: I would probably have used "Vänligen konfigurera..." but I will leave the choice to you here, the sentence is correct in its current form as well.

File: shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po (hunk 2 above)
Comment (maxg0) [inline shuup_stripe/locale/sv_SV/LC_MESSAGES/django.po]: How so?

**Pull request merged**